core/templates/dev/head/pages/exploration_player/PlayerServices.js

```diff
@@ -479,6 +479,12 @@ oppia.factory('ExplorationPlayerService', [
           StatsReportingService.recordSolutionHit(stateName);
         }
       },
+      recordLeftForRefresherExp: function(stateName, refresherExpId) {
+        if (!_editorPreviewMode) {
+          StatsReportingService.recordLeftForRefresherExp(
```

Comment:
Travis is reporting that this isn't a function. Looks like it was named as `recordLeaveForRefresherExp` in `StatsReportingService`